wasp-ai/src/client/pages/MainPage.jsx

```diff
@@ -67,19 +76,40 @@ const MainPage = () => {
     ],
     []
   );
-
   const [appAuthMethod, setAppAuthMethod] = useState(availableAuthMethods[0]);
 
   useEffect(() => {
+    if (userProjects?.length >= 1) {
+      checkIfUserStarredWasp({ username: user.username }).then((hasStarred) => {
+        if (!hasStarred) {
+          setIsAskForStarsModalOpen(true);
+        }
+      });
+    }
+  }, [userProjects, user]);
+
+  async function checkIfUserStarredWasp({ username }) {
+    const token = "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";
```

Comment:
@vincanger This is big no-no 😢 we should do the check on the backend to keep the token a secret.

**You should delete the leaked token right away.**

Reply:
yep. pushed the wrong commit too soon. I had it already refactored to be a query but forgot to commit the change before creating the PR